tests/ci/cdk/cdk/ssm/bm_framework_ssm_document.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+---
+schemaVersion: '2.2'
+description: aws-lc:bmFrameworkEc2Benchmark
+mainSteps:
+  - action: aws:runShellScript
+    name: runShellScript
+    inputs:
+      timeoutSeconds: '7200'
+      runCommand:
+        - sudo -i
+        - export DEBIAN_FRONTEND=noninteractive
+        - export CPU_TYPE=$(dpkg --print-architecture)
+        # if we want to disable CPU features, pass in the {OPENSSL_ia32cap} value
+        - if [ {OPENSSL_ia32cap} ]; then export OPENSSL_ia32cap={OPENSSL_ia32cap}; fi
+        - echo "${OPENSSL_ia32cap}"
+        # if we have a cpu type of x86, we want linux-x86
+        - if [ "${CPU_TYPE}" = amd64 ]; then export CPU_ARCH=linux-x86; fi
+        # if we have a cpu type of arm, we want linux-aarch
+        - if [ "${CPU_TYPE}" = arm64 ]; then export CPU_ARCH=linux-aarch; export IMAGE_TAG_POST=-aarch; fi
+        # create bm_framework directory and checkout aws-lc
+        - mkdir bm_framework
+        - cd bm_framework
+        - git clone {GITHUB_REPO} aws-lc-pr
+        - git clone https://github.com/awslabs/aws-lc.git aws-lc-prod
+        - git clone https://boringssl.googlesource.com/boringssl
+        - git clone https://github.com/openssl/openssl.git
```

Comment:
* Not need to checkout a specific commit of bssl/aws-lc?
* Nit: Move these git clone to `./aws-lc-pr/tests/ci/build_run_benchmarks.sh` so that the script can be executed standalone? Some variables can be set to be default value when not defined.

Reply:
I think we want to test our PR code against the tip of main of aws-lc. I'm unaware of any specific version of BoringSSL that we should be benchmarking against, so I also defaulted to tip of main. Let me know if there is a specific version of BoringSSL that I should be benchmarking against and I will change it to that.

Good idea for moving the git clones to the the script, I will do that

Reply:
> Let me know if there is a specific version of BoringSSL that I should be benchmarking against and I will change it to that.

At least, the commit should be print out for later tracking. Regarding the impact of invariant, talk to team?